packages/app-admin-users-cognito/src/plugins/userMenu/userInfo.tsx

```diff
@@ -73,7 +77,10 @@ export const UserInfo = () => {
     const { currentTenant, defaultTenant } = security.identity;
 
     let wrapper: any = { Component: Link, props: { to: "/account" } };
-    if (currentTenant && defaultTenant && currentTenant.id !== defaultTenant.id) {
+    if (
+        !accountRoute ||
```

Comment:
Account route is a path to the user "account" page. When you use Okta, Auth0, or any external IDP, you don't have an account view, because you can't edit a user from within the Admin app. However, the `UserInfo` menu item shows details of the user (email, displayName), and optionally renders a link to the "account" view. That's what this is all about.

Reply:
@adrians5j Did some refactoring on this.